jaqpotpy/jaqpot.py

```diff
@@ -5,9 +5,8 @@
 
 import numpy as np
 import pandas as pd
-from keycloak import KeycloakOpenID
-
 import jaqpotpy
+from keycloak import KeycloakOpenID
```

Comment:
Remove

Reply:
why? it is used in the script

Reply:
oops, I didn't see it in your code so I thought it's a mistake, resolving.